py/client/examples/import_test_data.py

```diff
@@ -10,18 +10,19 @@
 
 """
 
-from pyarrow import csv
+from pyarrow import parquet
 
-from examples.csv_downloader import download_csv
+from examples.file_downloader import download_file
 from pydeephaven import Session, Table
 from pydeephaven.utils import is_deephaven_compatible
 
 
 def import_taxi_records(dh_session: Session) -> Table:
 
-    # download the CSV data and read it into a pyarrow table and prepare it for uploading into DH
-    csv_file_name = download_csv(url="https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2020-12.csv")
-    pa_table = csv.read_csv(csv_file_name)
+    # download the parquet data and read it into a pyarrow table and prepare it for uploading into DH
+    parquet_file_name = download_file(url="https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-11.parquet",
+                                      file_name="sample.parquet", reuse_existing=True)
+    pa_table = parquet.read_table(parquet_file_name)
```

Comment:
I am not against the change but can't help wondering about the lifespan of the parquet file.